server/schemas/resolvers.js

```diff
@@ -40,113 +35,107 @@ const resolvers = {
-        });
-        return user.orders.id(_id);
+        console.log("HEREE");
+        const order = await Order.findById(_id);
+        console.log(order);
+        return order;
       }
       throw new AuthenticationError("Not logged in");
     },
-    checkout: async (parent, args, context) => {
-      const url = new URL(context.headers.referer).origin;
-      const order = new Order({ treatments: args.treatments });
-      const line_items = [];
-      const { treatments } = await order.populate("treatments");
-      // check !
-      for (let i = 0; i < treatments.length; i++) {
-        const treatment = await stripe.treatments.create({
-          name: treatments[i].name,
-          description: treatments[i].description,
-          images: [`${url}/images/${treatments[i].image}`],
-        });
-        const price = await stripe.prices.create({
-          treatment: treatment.id,
-          unit_amount: treatments[i].price * 100,
-          currency: "GBP",
+    // checkout: async (parent, args, context) => {
+    //   const url = new URL(context.headers.referer).origin;
+    //   const order = new Order({ treatments: args.treatments });
+    //   const line_items = [];
+    //   const { treatments } = await order.populate("treatments");
+    //   // check !
+    //   for (let i = 0; i < treatments.length; i++) {
+    //     const treatment = await stripe.treatments.create({
+    //       name: treatments[i].name,
+    //       description: treatments[i].description,
+    //       images: [`${url}/images/${treatments[i].image}`],
+    //     });
+    //     const price = await stripe.prices.create({
+    //       treatment: treatment.id,
+    //       unit_amount: treatments[i].price * 100,
+    //       currency: "GBP",
+    //     });
+    //     line_items.push({
+    //       price: price.id,
+    //       quantity: 1,
+    //     });
+    //   }
+    //   const session = await stripe.checkout.sessions.create({
+    //     payment_method_types: ["card"],
+    //     line_items,
+    //     mode: "payment",
+    //     success_url: `${url}/success?session_id={CHECKOUT_SESSION_ID}`,
+    //     cancel_url: `${url}/`,
+    //   });
+    //   return { session: session.id };
+  },
+  Mutation: {
+    addUser: async (parent, { name, email, password }) => {
+      const user = await User.create({ firstName, lastName, email, password });
+      const token = signToken(user);
+      return { token, user };
+    },
+    login: async (parent, { email, password }) => {
+      const user = await User.findOne({ email });
+      if (!user) {
+        throw new AuthenticationError("No user found with this email address");
+      }
+      const correctPw = await user.isCorrectPassword(password);
+      if (!correctPw) {
+        throw new AuthenticationError("Incorrect credentials");
+      }
+      const token = signToken(user);
+      return { token, user };
+    },
+    addUser: async (parent, { name, email, password }) => {
```

Comment:
shd del this query (line 94-98)? this is the old addUser, the updated one is in line 77-81